Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu
+
+# Install dependencies.
+RUN DEBIAN_FRONTEND=noninteractive apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
+     python3-pip python3-tk texlive-latex-base latexmk git emacs vim locales
```

Comment:
Where do we use `python3-tk`?

Reply:
It's needed for matplotlib on some systems IIRC.